data/rule_data.yml

```diff
@@ -6,6 +6,7 @@ rule_data:
   - registry.access.redhat.com/
   - registry.redhat.io/
   - brew.registry.redhat.io/rh-osbs/openshift-golang-builder
+  - nvcr.io/nvidia/cuda
```

Comment:
Is this something that will be needed by multiple products?

cc @rhartman93

Reply:
@lcarva as far as I know, https://github.com/openshift/instaslice-operator, requires it. But any product that needs to interact with nvidia GPU would need this.

Reply:
ack, makes sense. I think we just need some other folks to approve this change. Not sure if for Red Hat it would be better to just overwrite this on a per-product basis, or allow any product to use it (this PR).

I'll defer that to Ryan.

Reply:
Yeah, my opinion is that we should just add this to the product-specific rule data on a per-product basis. Adding it here is carte blanche.